go-controller/pkg/util/net_linux.go

```diff
@@ -375,7 +375,7 @@ func LinkNeighExists(link netlink.Link, neighIP net.IP, neighMAC net.HardwareAdd
 	return false, nil
 }
 
-func DeleteConntrack(ip string, port int32, protocol kapi.Protocol) error {
+func DeleteConntrack(ip string, port int32, protocol kapi.Protocol, fromOriginalTuple bool) error {
```

Comment:
I think it would be better to just make this new argument a netlink.ConntrackFilterType and make the caller pass in the filter type it wants to use.